src/interpreter_extra.cc

```diff
@@ -4663,8 +4663,31 @@ static void opGetPartyMember(Program* program)
 // 0x45C6DC
 static void opGetRotationToTile(Program* program)
 {
-    int tile2 = programStackPopInteger(program);
-    int tile1 = programStackPopInteger(program);
+    // CE: There is a bug in Olympus (tgrdqest) - object is passed as one of the
+    // arguments instead of tile. Original game (x86) does not distinguish
+    // between integers and pointers, so one of the tiles is silently ignored
+    // while calculating rotation. As a workaround this opcode now accepts
+    // both integers and objects.
```

Comment:
Maybe the bug should be fixed in the script?) Sounds like a rabbit hole allowing such ambiguity in arguments. If it says tile num, it should be tile num IMO.

Reply:
I agree, keep the original behavior (only accept int) but write warning in the debug log. This is a really bad idea to try and hack the engine around bugs in scripts :)